Foundation/NSString.swift

```diff
@@ -228,7 +228,12 @@ public class NSString : NSObject, NSCopying, NSMutableCopying, NSSecureCoding, N
     
     override internal var _cfTypeID: CFTypeID {
         return CFStringGetTypeID()
-    }
+  }
+  
+  public override func isEqual(object: AnyObject?) -> Bool {
```

Comment:
This probably could be split out into it's own request since it is relatively isolated from calendrical calculations